secret_hitler/app.py

```diff
@@ -345,39 +346,45 @@ async def invite(ctx, member : commands.MemberConverter):
         await sendBoard(game)
         await start_nomination(game)
     else:
-        embed = discord.Embed(title='Player joined the game', description='The player '+member.display_name+' joined the game! Waiting for more players', color=discord.Color.dark_red())
+        embed = discord.Embed(title="Player joined the game", description="The player "+member.display_name+" joined the game! Waiting for more players", color=discord.Color.dark_red())
         embed.add_field(name="Slots", value=str(len(game.players))+"/"+str(game.max_players))
         await client.get_channel(game.channel_id).send(embed=embed)
 
 
+@invite.error
+async def invite_error(ctx, error):
+    # Print the error message so the user knows why their invite failed
+    await ctx.send(str(error))
```

Comment:
Here's an example of a command-specific error handler. You have to decorate it with `@<command_name>.error`. I'm not sure if the name of the method has to match the pattern `<command_name>_error`, though. I just followed that pattern since it was what was in the documentation and seemed like a reasonable method name to me.